src/Assignment1/Program.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+
+namespace Assignment1
+{
+    /// <summary>
+    /// This is main class <see cref="Program"/>
+    /// </summary>
+    public class Program
+    {/// <summary>
+     /// This is main function
+     /// </summary>
+     /// <param name="args">This is a sample argument</param>
```

Comment:
```suggestion
     /// <param name="args">Sample argument</param>
```

Changle similarly for method arguments as well